pyccel/parser/semantic.py

```diff
@@ -1992,7 +2020,8 @@ def _visit_PyccelPow(self, expr, **settings):
 
     def _visit_MathSqrt(self, expr, **settings):
         func = self.scope.find(expr.funcdef, 'functions')
-        arg, = self._handle_function_args(expr.args, **settings)
+        args = self._handle_function_args(expr.args, **settings)
+        arg = args[0]
```

Comment:
I prefer the original code, because it was clear that `_handle_function_args` must return an iterable with only one element. Does Pylint complain about this too?

Reply:
In general this is a very useful warning! But here we know that the returned tuple has exactly 1 element, so there is no danger of a "Possible unbalanced tuple unpacking". The problem is that Pylint does not understand that `len(arg) == len(expr.args)` based on static code analysis.

I suppose that could make Pylint happy by writing this ugly code,
```python
arg, *_ = self._handle_function_args(expr.args, **settings)
```
but I recommend against this workaround which makes the code more obscure.

I actually prefer to disable Pylint on the specific lines where it complains about this.